ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlServerEndpoints.java

```diff
@@ -151,17 +156,20 @@ public KsqlServerEndpoints(
   }
 
   @Override
-  public CompletableFuture<EndpointResponse> executeQueryRequest(final KsqlRequest request,
+  public CompletableFuture<EndpointResponse> executeQueryRequest(
```

Comment:
Nit: Unnecessary whitespace changes make it harder to use blame.

Reply:
The formatting is off here, when we break arguments in multiple lines, the first one must be in a new line as well